modules/openapi-generator/src/main/resources/kotlin-spring/apiUtil.mustache

```diff
@@ -12,9 +12,9 @@ object ApiUtil {
     fun setExampleResponse(req: NativeWebRequest, contentType: String, example: String) {
         try {
             val res = req.getNativeResponse(HttpServletResponse::class.java)
```

Comment:
```suggestion
            val res: HttpServletResponse? = req.getNativeResponse(HttpServletResponse::class.java)
```

Reply:
> If `res` is nullable, how about declare `HttpServletResponse?` ?

kotlin infer itself that res is of type `HttpServletResponse?`